src/test/java/jalau/at18/katas/likes/sarai/MainTest.java

```diff
@@ -0,0 +1,11 @@
+package jalau.at18.katas.likes.sarai;
+
+import static org.junit.Assert.*;
+public class MainTest {
+    public void shouldReturnLikeThis() {
+        String input = "juan lulu jose";
+        String expected = "juan,lulu,jose, like this";
```

Comment:
This should match kata requirement (values can be anything: juan, lulu, jose, that's ok, but your expected value in test is missing "and" text and spaces after commas)
```
["Max", "John", "Mark"]           -->  "Max, John and Mark like this"
```

Also, you need to add test/asserts for all cases described in the kata requirement
```
[]                                -->  "no one likes this"
["Peter"]                         -->  "Peter likes this"
["Jacob", "Alex"]                 -->  "Jacob and Alex like this"
["Max", "John", "Mark"]           -->  "Max, John and Mark like this"
["Alex", "Jacob", "Mark", "Max"]  -->  "Alex, Jacob and 2 others like this"
```